web/ui/src/app/lineage/graph/graph-builder.ts

```diff
@@ -34,13 +37,74 @@ export function lineageToGraph(lineage: IDataLineage,
         visibleOperations: IOperation[] = operationsByVisibility.true,
         hiddenOperations: IOperation[] = operationsByVisibility.false,
         hiddenOpIds: string[] = _.map(hiddenOperations, "mainProps.id"),
-        visibleNodes = visibleOperations.map(op => new RegularVisNode(op)),
+        visibleNodes = visibleOperations.map(op => new RegularVisNode(op, getLabel(op, expressionRenderService))),
         visibleEdges = createVisibleEdges(lineage, hiddenOpIds)
 
     return new VisModel(
         new vis.DataSet<VisNode>(visibleNodes),
         new vis.DataSet<VisEdge>(visibleEdges))
 }
+/**
+ * Get label from an operation
+ * 
+ * @param operation the operation where the label should be retrieved
+ * @param expressionRenderService service to render expressions
+ * 
+ * @returns the label from the operation
+ */
+export function getLabel(operation : (IOperation), expressionRenderService : ExpressionRenderService){
+    let label = operation.mainProps.name + "\n"; 
```

Comment:
Semicolons. Unless you have a strong opinion why we _should_ use semicolons I'd say we rather don't.
While in JS world, there is a never ending battle about if semicolons are good or evil, in TS however they make even less sense. Even in JS the cases that make ASI to behave unexpectedly are rare and strange enough to not belong to a good coding style anyway. But in TS the compiler will catch a good chuck of those leaving even less room for ASI related mistakes. There is a good article about it - https://medium.com/@eugenkiss/dont-use-semicolons-in-typescript-474ccfe4bdb3

Reply:
I prefer semicollons in TS, but I  dont mind either way.

Reply:
As you guys prefer, I have no preference.